src/constants/misc.ts

```diff
@@ -1 +1,3 @@
 export const DEFAULT_BOARD_NAME = "scrumlr.io";
+export const MIN_PASSWORD_LENGTH = 1;
```

Comment:
It was 1 before, and I didn't want to touch the password requirements because I think it would be more appropriate to address this in a separate issue.